assets/_scss/components/_footer.scss

```diff
@@ -0,0 +1,299 @@
+.usa-footer {
+
+  button {
+    width: 100%;
+  
+    @media (min-width: $tablet-up) {
+      width: auto;
+    }
+  }
+
+  .usa-unstyled-list {
+    display: block;
+  }
+
+  .usa-footer-primary-link {
+
+    color: $color-base-link;
+    display: block;
+    font-weight: $font-bold;
+    margin-top: 0;
+    padding: {
+      bottom: 2rem;
+      top: 2rem;
+    }
+    text-transform: uppercase;
+
+    @media (min-width: $tablet-up) {
+      border-top: none;
+      font-size: 1.9rem;
+      text-transform: none;
+    }  
+  }  
+}
+
+.usa-footer a {
+  font-weight: normal;
+}
+
+.usa-footer-return-to-top {
+  padding: {
+    bottom: 2rem;    
+    top: 2rem;
+  }
+}
+
+.usa-footer-primary-section {
+  background-color: $color-grey-100;
+
+  .usa-footer-primary-content {
```

Comment:
If these classes are so specific, maybe they don't have to be nested, which will keep specificity low. I'll file an issue for it, it's not urgent.